### PR TITLE
Add code for evaluating AVA action detection performance.

### PR DESCRIPTION
**Pull request by raviddoss**
The main file is get_ava_performance.py. It uses code metrics evaluation code, stored under ava/, which was adapted from https://github.com/tensorflow/models/tree/master/research/object_detection by removing dependencies on tensorflow and protocol buffers.

To use this, you must provide the label map file ava/ava_action_list_v2.1_for_activitynet_2018.pbtxt.txt (included) as well as the csv file of ground truth action boxes and your detection action boxes (not included).

### CODE REVIEW

**Comment by escorciav** [comment]
Thanks, LGTM.

@raviddoss, please confirm the following
1. the score of the detection should be in the 8-th column of the CSV.
2. these warning messages are expected
```
WARNING:root:The following classes have no ground truth examples: [ 2 16 18 19 21 23 25 31 32 33 35 39 40 42 44 50 53 55 71 75]
/ActivityNet/Evaluation/ava/metrics.py:144: RuntimeWarning: invalid value encountered in true_divide
  num_images_correctly_detected_per_class / num_gt_imgs_per_class)
```

I tested it in the validation set with random boxes and the same number of instances than the ground truth.

**Comment by raviddoss** [comment]
Thanks @escorciav.

1. Yes, that's right. There are some minor clarifications required for the task description. Can I just send a pull request to update this web page https://github.com/activitynet/activitynet.github.io/blob/master/challenges/2018/tasks/guest_ava.html?
1.  I just committed an additional change to suppress the divide by zero warning (it was intended) and make "The following classes" message info rather than warn, since for the competition we're only testing on the 60 most common classes in AVA.
